persist/schema/types.go

```diff
@@ -31,14 +31,14 @@ const MajorVersion = 1
 
 // IndexInfo stores metadata information about block filesets
 type IndexInfo struct {
+	MajorVersion int64
```

Comment:
hm i thought the ordering was kept consistent between here and order of fields written out

Reply:
It was, but it seems silly considering it's not necessary and `MajorVersion` really should be first ha.  I can revert if you'd like.

Reply:
Nah i mean as long as we the order is captured in code w/e. This was a minor nicety but it does make the struct clunky